### DISCUSSION TITLE
Algolia AIS and Nuxt3

### DISCUSSION THREAD

**Discussion by raducretu** [nuxt/nuxt#17396] [Questions]
Has someone integrated Algolia AIS with SSR? 
Their documentation is not oriented towards setup, it's with mixins and etc...

[the remark this answers is not in the document]
**Reply by Baroshem** [reply]
Hey, the SSR is not working yet. We are experiencing some issues to make it work. It seems like the requester is not compatible with the newest version of Nuxt. 

You can follow this PR where we are testing things out

https://github.com/nuxt-community/algolia-module/pull/46

[the remark this answers is not in the document]
**Reply by pi0** [reply]
Thanks for quick reply! Is there an open issue to track?

[the remark this answers is not in the document]
**Reply by Baroshem** [reply]
Not yet, but I can create one
